src/pages/NewClanPage.tsx

```diff
@@ -257,103 +288,152 @@ function NewClanPage() {
-              type="radio"
-              id="winter-radio"
-              value="Leaf-bare"
-              name="season"
-            ></input>
-            <label htmlFor="winter-radio">Leaf-bare (Winter)</label>
-          </div>
+            <div className="radio-row">
+              <input
+                tabIndex={0}
+                type="radio"
+                id="winter-radio"
+                value="Leaf-bare"
+                name="season"
+                onChange={(e) => setSeason(e.target.value)}
+              ></input>
+              <label htmlFor="winter-radio">Leaf-bare (Winter)</label>
+            </div>
 
-          <div className="radio-row">
-            <input
-              tabIndex={0}
-              type="radio"
-              id="spring-radio"
-              value="Newleaf"
-              name="season"
-              defaultChecked
-            ></input>
-            <label htmlFor="spring-radio">Newleaf (Spring)</label>
-          </div>
+            <div className="radio-row">
+              <input
+                tabIndex={0}
+                type="radio"
+                id="spring-radio"
+                value="Newleaf"
+                name="season"
+                defaultChecked
+                onChange={(e) => setSeason(e.target.value)}
+              ></input>
+              <label htmlFor="spring-radio">Newleaf (Spring)</label>
+            </div>
 
-          <div className="radio-row">
-            <input
-              tabIndex={0}
-              type="radio"
-              id="summer-radio"
-              value="Greenleaf"
-              name="season"
-            ></input>
-            <label htmlFor="summer-radio">Greenleaf (Summer)</label>
-          </div>
+            <div className="radio-row">
+              <input
+                tabIndex={0}
+                type="radio"
+                id="summer-radio"
+                value="Greenleaf"
+                name="season"
+                onChange={(e) => setSeason(e.target.value)}
+              ></input>
+              <label htmlFor="summer-radio">Greenleaf (Summer)</label>
+            </div>
+          </fieldset>
+          <fieldset>
+            <legend>Variant</legend>
+            <div className="dropdown">
+              {
+                biomeVariants[biome.toLowerCase()].map((campName, index) => {
+                  return (
+                    <div className="radio-row">
+                      <input
+                        tabIndex={0}
+                        type="radio"
+                        id={`${campName}-variant-radio`}
+                        value={`camp${index+1}`}
+                        name="variant"
+                        defaultChecked={index == 0}
+                        checked={campNum == `camp${index+1}`}
+                        onChange={(e) => setCampNum(e.target.value)}
+                      ></input>
+                      <label htmlFor={`${campName}-variant-radio`}>{campName}</label>
```

Comment:
```suggestion
                      <label htmlFor={`${campName}-variant-radio`}>{campName.substring(0, 1).toUpperCase() + campName.substring(1)}</label>
```
Consistent with previous options.